api/src/test/java/org/pmiops/workbench/cdr/CdrVersionServiceTest.java

```diff
@@ -132,20 +139,7 @@ public void testSetCdrVersionDefault() {
   }
 
   @Test
-  public void testSetCdrVersionDefaultId() {
```

Comment:
Many of these tests were redundant, so I removed them.

Backstory: we previously had 2 ways to refer to a Default CDR Version, by DB object and by ID.  We have collapsed these into DB object only, and we mistakenly converted the "ID" tests to use the object instead of removing them.

Reply:
There's only one new test here, starting at 212